rama-pac/src/pac_fetcher/mod.rs

```diff
@@ -0,0 +1,21 @@
+use rama_core::{Context, error::BoxError};
+use rama_http::{Body, Request, Uri, service::client::HttpClientExt};
+
+pub async fn fetch_pac<W>(web_client: &W, pac_uri: &Uri) -> Result<String, BoxError>
+where
+    W: HttpClientExt,
```

Comment:
We do not trait bound on `HttpClientExt` instead trait bound on regular `Service`. `HttpClientExt` is implemented for `Service`.